storage/ndb/src/kernel/blocks/dbtc/DbtcMain.cpp

```diff
@@ -18102,6 +18104,7 @@ bool Dbtc::sendScanFragReq(Signal *signal, ScanRecordPtr scanptr,
   req->clientOpPtr = scanFragP.p->m_apiPtr;
   req->batch_size_rows = scanP->batch_size_rows;
   req->batch_size_bytes = scanP->batch_byte_size;
+  req->ttl_purge_window_size = scanP->m_ttl_purge_window_size;
```

Comment:
An alternative approach is to add an extra bit and only set this variable when it is actually used, this approach adds 32 bits of overhead for each SCAN_TABREQ which can affect performance a few percentage points in some cases.

Reply:
I see your point. Personally, I think adding one more field keeps things clearer and simpler, and 32 bits shouldn’t be a big deal performance-wise. But happy to go with your suggestion if we spot any regressions during benchmarking.